pkg/util/util.go

```diff
@@ -35,7 +37,15 @@ func GetEncryptionTransformers(encryptionConfigSecretName string, secrets v1core
 	if !ok {
 		return transformerMap, fmt.Errorf("no encryptionConfig provided")
 	}
-	return encryptionconfig.ParseEncryptionConfiguration(bytes.NewReader(encryptionConfigBytes))
+	err = ioutil.WriteFile("encryptionConfig.yaml", encryptionConfigBytes, os.ModePerm)
```

Comment:
I'd suggest adding a `defer` block to delete this file.

Reply:
True!